graphium/cli/train_finetune.py

```diff
@@ -44,8 +44,23 @@ def run_training_finetuning(cfg: DictConfig) -> None:
 
     st = timeit.default_timer()
 
+    # Disable wandb if the user is not logged in.
+    if wandb.login() is False:
+        logger.info(
+            "Not logged in to wandb - disabling wandb logging.\n"
+            + "To enable wandb, run `wandb login` from the command line."
+        )
+        wandb.init(mode="disabled")
+
```

Comment:
This piece of code is now a duplicate of L57 to L62, right? We should be able to delete it.  

```suggestion
```